herbert_core/classes/instrument_classes/@IX_map/product.m

```diff
@@ -0,0 +1,97 @@
+function obj_out = product (objB, objA)
+% Create the IX_map that gives the mapping of workspaces created by an second map
```

Comment:
same comment for identifying workspace and spectra concepts

Reply:
same comment from my side as above.